src/main/java/frc/robot/Constants.java

```diff
@@ -18,6 +18,13 @@ public static class IntakeConstants {
     public static final double INTAKE_PERCENTAGE = -0.75;
   }
 
+  public static class ShooterConstants {
+    public static final int SHOOTER_MOTOR_ID = 55;
+    public static final double SHOOTER_MINIMUM_SPEED = 150; // Placeholder; in RPM
```

Comment:
`SHOOTER_IDLE_SPEED` is a more descriptive name

Reply:
Resolved in 429dcbb